server/intelligence-service/app/model.py

```diff
@@ -78,29 +78,13 @@ def call_model(state: State):
 app = workflow.compile(checkpointer=memory)
```

Comment:
I would also remove `checkpointer=memory` for now. If we use `MemorySaver` we would also need `PostgresSaver` for a production environment as I understand and we are not there yet. What is it doing as of now?